wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/versions/Http.kt

```diff
@@ -0,0 +1,38 @@
+package it.krzeminski.githubactions.wrappergenerator.versions
+
+import it.krzeminski.githubactions.wrappergenerator.domain.ActionCoords
+import kotlinx.serialization.builtins.ListSerializer
+import kotlinx.serialization.json.Json
+import okhttp3.OkHttpClient
+import okhttp3.Request
+
+val ActionCoords.apiTagsUrl: String
+    get() = "https://api.github.com/repos/$owner/$name/git/matching-refs/tags/v"
+
+@kotlinx.serialization.Serializable
+data class GithubTag(
+    val ref: String,
+)
+
+val json = Json { ignoreUnknownKeys = true }
+
+val okhttpClient by lazy {
+    OkHttpClient()
+}
+
+fun ActionCoords.fetchAvailableVersions(GITHUB_TOKEN: String): List<String> {
+    val request: Request = Request.Builder()
+        .header("Authorization", "token $GITHUB_TOKEN")
+        .url(apiTagsUrl)
+        .build()
+
+    val content = okhttpClient.newCall(request).execute().use { response ->
+        if (response.isSuccessful.not()) {
```

Comment:
Is there any advantage of this notation over
```suggestion
        if (!response.isSuccessful) {
```
? Is `not()` more prominent in your opinion?

Reply:
I prefer the `not()` because it keeps a coherent left-to-right reading order.